<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1</class>
 <widget class="QMainWindow" name="MainWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计数器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PushButton_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>减</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>乘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>第一个数：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>第二个数：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>结果     :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
